single_kernel_mongo/core/abstract_upgrades.py

```diff
@@ -514,25 +483,6 @@ def is_cluster_healthy(self: GenericMongoDBUpgradeManager[MongoDBOperator]) -> b
         if self.state.is_sharding_component and not self.state.has_sharding_integration:
             return True
 
-        # It is possible that in a previous run of post-upgrade-check, that the unit was set to
```

Comment:
These were removed - as we perform more in depth status checks (i.e. can read/write), membership, etc.